android/app/src/main/java/com/zulipmobile/notifications/GCMPushNotifications.java

```diff
@@ -172,12 +180,13 @@ private static void updateNotification(
             builder.addAction(action);
         }
 
+        // Note: This code only set custom sound for API < 26 (from higher API sound is managed by channels)
         if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.LOLLIPOP) {
             AudioAttributes audioAttr = new AudioAttributes.Builder()
                     .setUsage(AudioAttributes.USAGE_NOTIFICATION).build();
-            builder.setSound(Uri.parse("android.resource://" + context.getPackageName() + "/" + R.raw.zulip), audioAttr);
+            builder.setSound(Uri.parse(ContentResolver.SCHEME_ANDROID_RESOURCE +"://" + context.getPackageName() + NOTIFICATION_SOUND_PATH), audioAttr);
         } else {
-            builder.setSound(Uri.parse("android.resource://" + context.getPackageName() + "/" + R.raw.zulip));
+            builder.setSound(Uri.parse(ContentResolver.SCHEME_ANDROID_RESOURCE +"://" + context.getPackageName() + NOTIFICATION_SOUND_PATH));
```

Comment:
I like the idea of deduplicating the `"/raw/zulip"` string. Let's take that farther -- let's move the whole `Uri.parse(...)` call into a little static method that returns a `Uri`.

That will also let us bring the `"/raw/zulip"` string constant back into that code -- the code is easier to read if it doesn't refer to a static constant defined 100 lines up in the file.